api/answer.py

```
@@ -236,67 +245,143 @@ def query(self,q_info:dict) -> Optional[str]:
+                logger.info(f"从缓存中获取答案：{q['title']} -> {answer}")
+                results[idx] = answer.strip()
+            else:
+                pending_indices.append(idx)
+
+        if not pending_indices:
+            return results
+
+        sub_q_list = [q_list[idx] for idx in pending_indices]
+        sub_results = self._query_all(sub_q_list, query_delay=query_delay)
+
+        for idx, ans in zip(pending_indices, sub_results):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)